.scss-lint.yml

```diff
@@ -97,7 +97,7 @@ linters:
 
   PropertySpelling:
     enabled: true
-    extra_properties: ['text-size-adjust', 'composes']
+    extra_properties: ['text-size-adjust', 'composes', 'menuBreakpoint']
```

Comment:
I‘m using `
:export {
    menuBreakpoint: $menu-breakpoint;
 }
` at some point in the CSS. I can then access that variable from the JS, which is cool because it means I don‘t have to manually sync the variables if I ever change them, only changing them in the CSS is enough.
But because the linter considers `menuBreakpoint` as a real property, it was triggering an error. So I ignored it here :)